README.md

```diff
@@ -1 +1,43 @@
-# HelloMicroverse
\ No newline at end of file
+# HelloMicroverse
+
+
+My first prject at Microverse program
+
+## Built With
+
+- Major languages Html and Css
+- Frameworks
+- Technologies used
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
+
+## Authors
+
```

Comment:
- Kindly give general information about your project, how to use it and how to run it locally and another things that you want to give for non technical people.